pocs/utils/rs232.py

```diff
@@ -142,27 +156,26 @@ def write(self, value):
 
         return response
 
-    def read(self):
-        """
-        Reads value using readline
-        If no response is given, delay and then try to read again. Fail after 10 attempts
+    def read(self, retry_limit=5, retry_delay=0.5):
+        """Reads next line of input using readline.
+
+        If no response is given, delay for retry_delay and then try to read
```

Comment:
Better than previous docstring, but we have explicit Args section. Need to start enforcing docstring standards. I have a plugin for Sublime that fills out all the boilerplate.

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google